### ISSUE TITLE
redis_config client doesn't work 

### ISSUE THREAD

**Issue by Dinaamagdy** [threefoldtech/jumpscaleX_core#453]
## Steps
- cl=j.clients.redis_config.get(name="test", port=10000)                     
- cl.redis     
```
Wed 22 14:38:55 onfig/RedisConfig.py - 111 - redis                              : EXCEPTION: 
    TypeError("get() got an unexpected keyword argument 'password'",)
--TRACEBACK------------------
<stdin> in <module>
    1    
/sandbox/lib/jumpscale/Jumpscale/clients/redisconfig/RedisConfig.py in redis
    111    set_patch=self.set_patch,
-----------------------------

get() got an unexpected keyword argument 'password'
```

**Comment by xmonader** [comment]
fixed in f9b91d74371a19ee132bb09d54913ecd6edfc3fd

**Comment by Dina-Abd-Elrahman** [comment]
- verified.
